core/settings.py

```diff
@@ -168,3 +169,17 @@
         },
     },
 }
+
+# Sentry Configuration
+sentry_sdk.init(
+    # dsn="https://1936689219fbec45b1437ee54bcd2a29@o345774.ingest.us.sentry.io/4507181591101440",
```

Comment:
I think this should be removed from the commit if it allows anyone to publish to our sentry?